asmcli/lib/parse.sh

```diff
@@ -473,3 +473,157 @@ arg_required() {
     fatal "Option ${1} requires an argument."
   fi
 }
+
+x_parse_install_args() {
+  if [[ "${*}" = '' ]]; then
+    x_usage >&2
+    exit 2
+  fi
+
+  # shellcheck disable=SC2064
+  trap "$(shopt -p nocasematch)" RETURN
+  shopt -s nocasematch
+
+  REVISION_LABEL_REGULAR="asm-managed"; readonly REVISION_LABEL_REGULAR
+  REVISION_LABEL_RAPID="asm-managed-rapid"; readonly REVISION_LABEL_RAPID
+  REVISION_LABEL_STABLE="asm-managed-stable"; readonly REVISION_LABEL_STABLE
```

Comment:
These don't change depending on experimental or not, can we just add them to init()?

Reply:
Done